driver-rabbitmq/src/main/java/io/openmessaging/benchmark/driver/rabbitmq/RabbitMqBenchmarkDriver.java

```diff
@@ -65,7 +65,7 @@ public void initialize(File configurationFile, StatsLogger statsLogger) throws I
 
     @Override
     public void close() throws Exception {
-        if (connection != null) {
+        if (connection != null && connection.isOpen()) {
```

Comment:
`isOpen()` may cause thread BLOCKED as it lock a `monitor` object inside the method.
I recommend just simply close the connection without `connection.isOpen`
https://www.rabbitmq.com/api-guide.html#disconnecting

Reply:
Indeed. I had added it because `connection.close()` can produce `ShutdownSignalException`.
But it seems it's better to just catch `ShutdownSignalException`.
There's also a `channel.isClosed()` check done when closing the consumers/producers.
I'll do a PR to fix all these.